app/assets/javascripts/modules/ukvi-ab-test-september-2016.js

```diff
@@ -0,0 +1,57 @@
+//= require govuk/multivariate-test
+
+var UkviABTest = UkviABTest || {};
+
+UkviABTest.isSection = function(pathname) {
+  return window.location.pathname === pathname;
+}
+
+UkviABTest.AddtoApplySection = function() {
+  var html = "<p>You can <a " +
+   "href='https://visas-immigration.service.gov.uk/product/eea-qp'>apply online "+
+   "as a qualified person</a> but not if you’re a student or self-sufficient "+
+    "person and you're either:"+
```

Comment:
The paragraph here needs to be closed </p> before the opening unordered list <ul>, as list elements cannot be children of p elements.